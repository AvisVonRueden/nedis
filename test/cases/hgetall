*4
$4
HSET
$6
user:0
$4
name
$2
tj
-----------
*4
$4
HSET
$6
user:0
$5
email
$11
tj@test.com
-----------
*2
$7
HGETALL
$6
user:0
