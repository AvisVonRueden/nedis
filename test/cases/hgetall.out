:1
:1
*4
$4
name
$2
tj
$5
email
$11
tj@test.com
